src/test/java/net/datafaker/sport/CricketTest.java

```diff
@@ -0,0 +1,29 @@
+package net.datafaker.sport;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class CricketTest extends SportFakerTest {
+
+    @Test
+    public void teams() {
+        assertThat(faker.cricket().teams()).isNotEmpty();
+    }
+
+    @Test
+    public void players() {
```

Comment:
```suggestion
    void players() {
```
for junit5 it could be hardened to packaged level